daisy_integration_tests/scripts/post_translate_test.sh

```diff
@@ -15,6 +15,19 @@
 
 # Basic image tests to validate if Linux image translation was successful.
 
+
+# To disable checking for the OS config agent, add an instance metadata
+# value of osconfig_not_supported: true.
+url=http://metadata.google.internal/computeMetadata/v1/instance/attributes/osconfig_not_supported
+if command -v curl; then
+  OSCONFIG_NOT_SUPPORTED="$(curl --header "Metadata-Flavor: Google" $url)"
+elif command -v wget; then
+  OSCONFIG_NOT_SUPPORTED="$(wget --header "Metadata-Flavor: Google" -O - $url)"
+else
+  echo "TestFailed: Either curl or wget is required to run test suite."
```

Comment:
i see a `fail` function is used elsewhere in this file. why not here?

Reply:
The `fail` function accumulates test failures, but allows the suite to continue. In this case, if we fail to read from the metadata server, then we have insufficient information to proceed with testing, and exit immediately.